CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## Unreleased
+## [2.9.0](https://github.com/idealista/cookiecutter-ansible-role/tree/2.9.0)
+### [Full Changelog](https://github.com/idealista/cookiecutter-ansible-role/compare/2.8.1...2.9.0)
+### Changed
+- *[#86](https://github.com/idealista/cookiecutter-ansible-role/issues/86) Upgrade molecule config to test under debian bullseye* @sperez
```

Comment:
Remember to use your github user :)

Reply:
Fixed with commit https://github.com/idealista/cookiecutter-ansible-role/pull/90/commits/fdbc7c3908f3013606fad0b1af3a41524eb2e1a7